_includes/default_roles/crier.md

```diff
@@ -1,19 +1,26 @@
-The _Crier_ reminds people in _Slack_ of the upcoming meeting.
+A _Crier_ reminds us in Slack of upcoming meetings.
 
-**1 Business Day Before Meeting**
+**One Business Day Before The Meeting**
 
-* [ ] Announce the meeting in our [Slack][chat]{:target="_blank"}.
-* [ ] Use the `Share` arrow icon to share that announcement with the [#general] channel.
+* [ ] *Announce* the meeting in our [Slack][chat]{:target="_blank"}.
+* [ ] *Send the announcement* to [#general] using the `Share` arrow.
 
 <img src="https://user-images.githubusercontent.com/9609562/220438340-2fed944a-142b-4217-bcae-5c0e0110ed05.png" width="500px" />
 
+**Behaviors**
+
+* When possible, move action items into a team backlog.  
```

Comment:
What is this for/mean?

Reply:
> What is this for/mean?

It needs work, but here's [the context](https://innersourcecommons.slack.com/archives/C04DT6NQX7G/p1681155861484859).

_I believe putting these small tasks in a board rather than having a Scribe follow up with people makes sense._

Reply:
Per our discussion, this was a wire crossing with the scribe role.

We also agreed this behavior may result in less follow-up.